packages/backend/src/helpers/flow-templates.ts

```diff
@@ -0,0 +1,88 @@
+import type { AppEventKeyPair } from '@plumber/types'
+
+import type {
+  CreateFlowTemplateInput,
+  DemoVideoDetails,
+  FlowConfig,
+} from '@/graphql/__generated__/types.generated'
+import Flow from '@/models/flow'
+import User from '@/models/user'
+
+import logger from './logger'
+
+export const DEFAULT_FLOW_TEMPLATE: CreateFlowTemplateInput = {
+  flowName: 'Send notifications',
+  trigger: {
+    appKey: 'formsg',
+    eventKey: 'newSubmission',
+  },
+  actions: [
+    {
+      appKey: 'postman',
+      eventKey: 'sendTransactionalEmail',
+    },
+  ],
+  demoVideoDetails: {
+    url: 'https://demo.arcade.software/FzpL1zCmibw0oXR6HUJi?embed&show_copy_link=true',
+    title: 'Send notifications',
+  },
+}
+
+async function makeFlowTemplate(
+  flowName: string,
+  trigger: AppEventKeyPair,
+  actions: AppEventKeyPair[],
+  user: User,
+  isPreCreated: boolean,
+  demoVideoDetails: DemoVideoDetails,
+): Promise<Flow> {
+  const { appKey: triggerAppKey, eventKey: triggerEventKey } = trigger
+
+  // transaction will insert flow and steps
+  return await Flow.transaction(async (trx) => {
+    const flowConfig: FlowConfig = {
+      demoConfig: {
+        onFirstLoad: true,
+        isPreCreated,
+        videoDetails: demoVideoDetails,
```

Comment:
i think it's better to map the videoUrls to an ID so that it can be easily replaced.
with this implementation, if we want to replace the video url for this demo pipe, we need to run a query to replace all occurences in the db.

however, if u store just the video id, such as `DEMO_SEND_NOTIFICATIONS` and then store the mapping on the frontend (perhaps), we can simply replace the URL in the file

Reply:
good catch sir thks